src/host/api.rs

```diff
@@ -154,11 +154,11 @@ pub mod networking {
         pub fn clone_udp_socket(udp_socket_id: u64) -> u64;
         pub fn tcp_flush(tcp_stream_id: u64, error_id: *mut u64) -> u32;
         pub fn tls_flush(tcp_stream_id: u64, error_id: *mut u64) -> u32;
-        pub fn set_read_timeout(tcp_stream_id: u64, duration: u64) -> u32;
+        pub fn set_read_timeout(tcp_stream_id: u64, duration: u64) -> ();
```

Comment:
You don't need the `-> ()` part, it's implicit, if a function doesn't return anything.